src/region.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! Mostly used for translating the Region enum to a string AWS accepts.
 //!
-//! For example: UsEast1 to "us-east-1"
+//! For example: `UsEast1` to "us-east-1"
```

Comment:
Why does clippy suggest removing backticks in doc comments? This change seems to be for the worse.